finding_data.md

```diff
@@ -19,6 +19,46 @@ You can change the order of the results using the 'Sort by' drop-down.
 
 ![Sort dropdown](images/finding_data_sort.png)
 
+### Keyword search tips 
+
+When using keyword search, you can use advanced operators to enhance your query.
+The examples below show how you can use those operators.
+
+* ```"armed forces" expenses``` - search for exact phrases by putting quote marks around them
+
+* ```primary school -care age``` - exclude words or quoted phrases by using a minus sign or NOT
+
+* ```primary OR secondary school``` - use 'OR' to allow alternatives (the default is to 'AND' all of the terms)
+
```

Comment:
I suggest you add: The full syntax is explained in the Lucene documentation https://lucene.apache.org/core/4_0_0/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#minitoc-area